doc/nrf/app_dev/device_guides/nrf54l/fota_update.rst

```diff
@@ -0,0 +1,236 @@
+.. _ug_nrf54l_developing_ble_fota:
+
+FOTA updates on nRF54L Series devices
+#####################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+.. fota_upgrades_intro_start
+
+|fota_upgrades_def|
+You can also use FOTA updates to replace the application.
+See the :ref:`app_dfu` page for general Device Firmware Update (DFU) information, such as supported methods for sending and receiving updates on the device.
+
+.. note::
+   For more information about introducing an upgradable bootloader, refer to :ref:`ug_bootloader_adding_sysbuild_upgradable`.
+
+.. fota_upgrades_intro_end
+
+.. _ug_nrf54l_developing_ble_fota_steps:
+
+FOTA over Bluetooth Low Energy
+******************************
+
+.. fota_upgrades_over_ble_intro_start
+
+FOTA updates are supported using MCUmgr's Simple Management Protocol (SMP) over Bluetooth®.
+The application acts as a GATT server and allows the connected Bluetooth Central device to perform a firmware update.
+To use FOTA over Bluetooth LE, samples must support Bluetooth peripheral role (:kconfig:option:`CONFIG_BT_PERIPHERAL`).
+
+The application supports SMP handlers related to:
+
+* Image management.
+* Operating System (OS) management used to reboot the device after the firmware upload is complete.
+* Erasing settings partition used to ensure that a new application is not booted with incompatible content in the settings partition written by the previous application.
```

Comment:
unrelated to this PR but we need to do something about this, settings should not be incompatible with different firmware versions, or even different firmwares, if using settings then only the keys which have been set will be used and the rest will be ignored so clearing settings *by default* is not needed and has been reported by a lot of users as causing issues, it should really be opt-in rather than opt-out

Reply:
maybe keys-name should be versioned. Agree that it is serous issue.